mesonbuild/interpreter/interpreter.py

```diff
@@ -604,6 +604,9 @@ def func_import(self, node: mparser.BaseNode, args: T.Tuple[str],
         # Some tests use "unstable_" instead of "unstable-", and that happens to work because
         # of implementation details
         if modname.startswith('unstable'):
+            if modname.startswith('unstable_'):
+                FeatureDeprecated.single_use('Importing unstable modules as "unstable_*"', '0.64.0', self.subproject,
+                                             'use "unstable-" instead.', node)
```

Comment:
This is not a Feature, just emit 

```
mlog.deprecation('Importing unstable modules as "unstable_*" was never supposed to work, '
                 'use "unstable-*" instead. This will be removed in a future Meson release.')
```